db/migrate/20231116142411_add_what_happens_next_markdown_to_forms.rb

```diff
@@ -0,0 +1,23 @@
+class AddWhatHappensNextMarkdownToForms < ActiveRecord::Migration[7.0]
+  def change
+    add_column :forms, :what_happens_next_markdown, :text
+
+    reversible do |direction|
+      Form.find_each do |form|
+        if form.made_live_forms.present?
```

Comment:
Why do we need to use forms to get to live versions? I think we should just go through all the live versions
pseudo  code changes so we dont just end up with the last 
```suggestion
      MadeLiveForm.find_each do |form|
```

Reply:
@aliuk2012 I copied it from a previous migration (`20230727151012_update_organisation_id_for_made_live_forms.rb`). I've updated it now though